crates/compiler/test_syntax/tests/test_fmt.rs

```diff
@@ -1693,31 +1693,32 @@ mod test_fmt {
             ),
         );
 
-        expr_formats_to(
-            indoc!(
-                r#"
-                    result = func arg
-                        [
-                            1,
-                            2,
-                            3,
-                        ]
+        // TODO: do we want to override the user's intent like this?
+        // expr_formats_to(
+        //     indoc!(
+        //         r#"
+        //             result = func arg
+        //                 [
+        //                     1,
+        //                     2,
+        //                     3,
+        //                 ]
 
-                    result
-                "#
-            ),
-            indoc!(
-                r#"
-                    result = func arg [
-                        1,
-                        2,
-                        3,
-                    ]
+        //             result
+        //         "#
+        //     ),
+        //     indoc!(
+        //         r#"
+        //             result = func arg [
+        //                 1,
+        //                 2,
+        //                 3,
+        //             ]
 
-                    result
-                "#
-            ),
-        );
+        //             result
+        //         "#
+        //     ),
+        // );
```

Comment:
Here and the test below: is there a reason to comment out this test? Agreed re. the concern on the todo, just worried about not introducing regressions while that question is discussed.

Reply:
Excellent question. If I'm being honest, this is because I broke this test when refactoring the Expr::Apply formatter. I was feeling lazy and thought I'd leave it for later ;)

Now that I have a few spare minutes, let me see about actually fixing this...